PWGHF/TableProducer/candidateCreator3Prong.cxx

```diff
@@ -85,6 +85,7 @@ struct HfCandidateCreator3Prong {
   Configurable<bool> createDs{"createDs", false, "enable Ds+/- candidate creation"};
   Configurable<bool> createLc{"createLc", false, "enable Lc+/- candidate creation"};
   Configurable<bool> createXic{"createXic", false, "enable Xic+/- candidate creation"};
+  Configurable<bool> createDstarToPiKPiBkg{"createDstarToPiKPiBkg", false, "enable D* candidate creation"};
```

Comment:
To avoid a new configurable and considering that this channel doesn't imply to create candidates that have a different flag than the D+ from the track index skim creator (which was the original idea behind the `create` configurables, to reduce the CPU time in Pb-Pb) I would simply enable the MC matching of D* everytime the D+ is enabled. Also, we could consider to do the MC matching for all the channels regardless the `create` configurables (which in the end arr important to pre-select the candidates in the spirit of reducing the CPU time)

Reply:
I'm not sure I follow the reasoning behind removing the `create` flags. Enabling matching only for analysed species still does save CPU, especially in `candidateCreatorMcGen.cxx` when we definitely not need all species to be matched.